src/Common/src/System/Runtime/InteropServices/FixedString.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO
```

Comment:
Still need a place to stick the methods- there are a lot of places we'd want to use these.